WordPress/WordPressTest/BlockEditorSettingsServiceTests.swift

```diff
@@ -10,15 +10,16 @@ class BlockEditorSettingsServiceTests: XCTestCase {
     private let blockSettingsThemeJSONResponseFilename = "wp-block-editor-v1-settings-success-ThemeJSON"
 
     private var service: BlockEditorSettingsService!
-    private var contextManager: TestContextManager!
-    private var context: NSManagedObjectContext!
+    private var contextManager: ContextManagerMock!
+    private var context: NSManagedObjectContext! {
+        contextManager.mainContext
+    }
```

Comment:
Given this is now a computed variable, and `mainContext` is not an implicitly unwrapped `Optional`, this shouldn't need to be either?

```suggestion
    private var context: NSManagedObjectContext {
        contextManager.mainContext
    }
```

In fact, later in the diff there are a few similar definitions that don't use the IUO.

Reply:
I probably forgot to delete the `!` when updating a few places. I'll update this one here and have a look at other places too.